.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Template Generation
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install aws-sam-cli==1.27.2
+        python -m pip install pytest~=6.0
```

Comment:
Will it install the latest version if we don't specify the version here? Do we need to update it every time we do a release?

Reply:
for `sam-cli`, we want to fix the version, otherwise when we update the template generation logics there, our template PR check will fail

Reply:
as for pytest, fixing the major version seems reasonable.

Reply:
> Do we need to update it every time we do a release?

Only update when we need to, like major change happens in sam-cli.